rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -162,11 +166,14 @@ without guessing. The format of the value may depend on the `ppt_scheme` attribu
 | Attribute      | Required? | Value                       |
 |----------------|-----------|-----------------------------|
 | ppt_scheme     | Y         | wamp.eth                    |
-| ppt_serializer | N         | cbor, flatbuffers           |
+| ppt_serializer | N         | cbor, flatbuffers, native*  |
```

Comment:
wamp.eth doesn't allow "native" for ppt_serializer

Reply:
Well is it not possible to use the same session serializer for creating the binary representation of the payload? The only limitation here is not to use json :) as it doesn't support binary types.
Maybe value `native` doesn't looks good here. As in any case payload must be serialized, but if this option is omitted then session serializer will be used (if it supports binaries)

Reply:
Oh... Good point: 
> this is a new meaning for "native": use the same serializer as in the enclosing WAMP session - which doesn't make sense, as there are at least 2 such sessions

So yeah, serializer needs to be specified explicitly, even if client wants to use the same one as session

Reply:
> So yeah, serializer needs to be specified explicitly, even if client wants to use the same one as session

yes, exactly, and the reason is: the serializer is applied _before_ e2ee, and hence it cannot be translated by the router - and hence the originator would need to request one, and the responder obeying this (and support it in the first place) - however, with PubSub, the responder actually is the publisher, and the publisher doesn't want to serialize multiple times for each subscriber serializer.

IOW: I think it makes more sense to let each peer specify the serializer it did use when sending a message ...